Tiltfile

```diff
@@ -56,7 +56,7 @@ LOOKUP_TABLE_ADDRESSES=[
 local_resource(
     "initialize-programs",
     """
-    pnpm turbo run:initialize-chain-id -- -u l -k ./tilt/keypairs/faucet.json --chain-id localnet &&
+    pnpm turbo run:initialize-chain-id -- localnet -u l -k ./tilt/keypairs/faucet.json &&
```

Comment:
nit: can you put the position argument localnet after the `-u l -k ./tilt/keypairs/faucet.json`